subprojects/packagefiles/libxslt/meson.build

```diff
@@ -97,6 +97,15 @@ endif
 
 libxml2_dep = dependency('libxml-2.0', version: '>=2.9.10')
 with_modules = libxml2_dep.get_variable(pkgconfig: 'modules', internal: 'modules', default_value: '0').to_int() != 0
+if host_machine.system() == 'cygwin'
+  config.set('MODULE_EXTENSION', '.cygdll')
+elif host_machine.system() == 'darwin'
+  config.set('MODULE_EXTENSION', '.bundle')
```

Comment:
Shouldn't this be `.dylib`?

Reply:
I have no idea. This was copy/paste from another wrap (htslib)

Reply:
`.bundle` applies to _directories_ with a particular layout, not to files.  On macOS, libtool at least thinks modules (runtime-loadable shared libraries) should be `.so` and other dynamic libraries should be `.dylib`.  Upstream libxslt's Autoconf selects `.so` but I'm not completely clear what we should do here.

On the other hand, upstream 1.1.43 disables plugins by default, with the future intent to remove them.

Reply:
Removed. This is based on cmake conf btw. I have no idea what cmake does.